code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1649,6 +1649,7 @@
 			baconbeacon.digest_delay()
 
 
+var/lastmonkeycubeexpand = 0

```

Comment:
well, the method in normal programming language would be a static variable in the function scope, but i do believe the closest parallel to it in byond is the global variable. If i am wrong i will gladly edit it